__tests__/components/editor/ValueConstraints.test.js

```diff
@@ -0,0 +1,93 @@
+import React from 'react'
+import { mount } from 'enzyme';
+import ValueConstraints from '../../../src/components/editor/ValueConstraints'
+
+const jsdom = require("jsdom");
+
+setUpDomEnvironment();
+
+const vcProps = {
+  "propertyTemplates": [
+    {
+      "valueConstraint": {
+        "valueTemplateRefs": [
+          "profile:bf2:Monograph:Work"
+        ],
+        "useValuesFrom": [
+          "http://id.loc.gov/foo/works"
+        ],
+        "valueDataType": {
+          "dataTypeURI": "http://id.loc.gov/ontologies/bibframe/Instance"
+        },
+        "defaults": [
+          {
+            "defaultURI": "http://id.loc.gov/foo/instance/mono",
+            "defaultLiteral": "foo"
+          }
+        ],
+        "mandatory": "false",
+        "repeatable": "true",
+        "editable": "true"
+      }
+    }
+  ]
+}
+
+const valueConstraints = (
+  <ValueConstraints
+    valueTemplateRefs = {vcProps.propertyTemplates[0].valueConstraint.valueTemplateRefs}
+    useValuesFrom = {vcProps.propertyTemplates[0].valueConstraint.useValuesFrom}
+    valueDataType = {vcProps.propertyTemplates[0].valueConstraint.valueDataType}
+    repeatable = {vcProps.propertyTemplates[0].valueConstraint.repeatable}
+    editable = {vcProps.propertyTemplates[0].valueConstraint.editable}
+    defaults = {vcProps.propertyTemplates[0].valueConstraint.defaults} >
```

Comment:
per actual code comment, is this a good place to use the spread operator to pass in props?

Reply:
👌